airbyte-webapp/src/packages/cloud/views/users/AccountSettingsView/components/PasswordSection.tsx

```diff
@@ -1,17 +1,44 @@
-import React from "react";
+import React, { useMemo } from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 import { Field, FieldProps, Form, Formik } from "formik";
+import styled from "styled-components";
 
 import {
   Content,
   SettingsCard,
 } from "pages/SettingsPage/pages/SettingsComponents";
 import { FieldItem } from "packages/cloud/views/auth/components/FormComponents";
 import { LabeledInput } from "components/LabeledInput";
+import { Button } from "../../../../../../components/base/Button";
+import { useMutation } from "react-query";
+import { useAuth } from "../../../../../firebaseReact";
+import { GoogleAuthService } from "../../../../lib/auth/GoogleAuthService";
+
+const ChangeEmailFooter = styled.div`
+  display: flex;
+  align-items: center;
+  height: 50px;
+`;
 
 export const PasswordSection: React.FC = () => {
   const formatMessage = useIntl().formatMessage;
 
+  const auth = useAuth();
+  const authService = useMemo(() => new GoogleAuthService(() => auth), []);
+
+  const { isLoading: isChangingPassword, mutate: changePassword } = useMutation<
+    void,
+    Error,
+    { currentPassword: string; password: string }
+  >(
+    async ({ password, currentPassword }) => {
+      return authService.changePassword(currentPassword, password);
```

Comment:
this should call `services/auth` service. We do not want this logic to spread everywhere. I suppose you need to add proxy to this method in AuthContext